### ISSUE TITLE
ActiveType 2.0 breaks rails_ops model operations

### ISSUE THREAD

**Issue by radioelephant** [sitrox/rails_ops#22]
### Steps to reproduce
1. Create rails app
2. Install rails_ops including controller mixin
3. Run any model operation (did not test load because it is usually not being run)

Example given:
```ruby
  # POST /users
  def create
    op Operations::User::Create

    if run
      [...]
    end
  end
```

### Error
```
ActiveType::MutationAfterCastError (Changing a record that has been used to create an ActiveType::Record could have unexpected side effects!):

app/operations/user/create.rb:16:in `perform'
app/controllers/users_controller.rb:21:in `create'
```

### Temporary Solution
Lock active_type to 1.10.1:
```bash
echo "gem 'active_type', '~> 1.10.1'" >> ./Gemfile
bundle update active_type
```

### Relevant Links
[Active_type#418](https://github.com/makandra/active_type/issues/148)
[Latest active_type release pre 2.0.](https://rubygems.org/gems/active_type)

**Comment by remofritzsche** [comment]
Thanks @radioelephant, will have a look at it 👍

**Comment by Adrian-Hirt** [comment]
Is fixed with release 1.1.18, thanks for the issue report. Let me know if anything else pops up, and if it's all good then you can close the issue.

**Comment by Adrian-Hirt** [comment]
Closing the issue as confirmed with @radioelephant IRL.
